### PR TITLE
Fixed camel case issue #457

### PR DESCRIPTION
**Pull request by rayko**
Issue with camel case endpoints [https://github.com/ruby-grape/grape-swagger/issues/457](https://github.com/ruby-grape/grape-swagger/issues/457)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 * [#444](https://github.com/ruby-grape/grape-swagger//pull/444): Default value provided in the documentation hash, override the grape default [@scauglog](https://github.com/scauglog).
 * [#443](https://github.com/ruby-grape/grape-swagger/issues/443): Type provided in the documentation hash, override the grape type [@scauglog](https://github.com/scauglog).
 * [#454](https://github.com/ruby-grape/grape-swagger/pull/454): Include documented Hashes in documentation output - [@aschuster3](https://github.com/aschuster3).
+* [#457](https://github.com/ruby-grape/grape-swagger/issues/457): Issue with camel case endpoints - [@rayko](https://github.com/rayko/).
```
Comment (dblock) [inline CHANGELOG.md]: Describe what the issue is (something like "camel case endpoint causes ...").

**Comment by dblock** [comment]
The build has failed, please check it out.

**Comment by rayko** [comment]
Is that better?

There was a failing spec before I put my hands on it: spec/swagger_v2/api_swagger_v2_detail_spec.rb:144

Tried with several ruby versions including the one used in the builder, just in case, but it still fails. I don't know what might be causing it, though it's just a \n missing on a returned value.

File: lib/grape-swagger.rb [hunk 2]
```diff
@@ -65,7 +65,6 @@ def combine_routes(app, doc_klass)
           next unless route_match
           resource = route_match.captures.first
           next if resource.empty?
-          resource.downcase!
```
Comment (LeFnord) [inline lib/grape-swagger.rb]: maybe you can re-use this method [operation_id.rb#L13](https://github.com/ruby-grape/grape-swagger/blob/master/lib/grape-swagger/doc_methods/operation_id.rb#L13)

**Comment by LeFnord** [comment]
@Bugagazavr …removed bug label, cause it isn't one, by using ruby naming conventions 😉

**Comment by dblock** [comment]
Is that spec failing on HEAD?

**Comment by dblock** [comment]
Looks like a newer version of kramdown/rouge is generating slightly different text. Go ahead and just fix the spec (change the `\n` in there) and make the build green and we can merge.

**Comment by rayko** [comment]
Should I include a comment in the changelog about that? Also, according to @LeFnord this is not a bug anymore, so my previous update on the changelog might need to be moved to the feature section instead of leaving it in the fixes section.

**Comment by dblock** [comment]
No need to changelog spec fixes. 

Is this really not a bug? Remove your fix, does the spec pass? If so it just doesn't need a CHANGELOG, but we should keep the spec.

**Comment by rayko** [comment]
It's green now. Rebel spec aside, this addresses the issue I've opened earlier, but the labels were changed, so I was asking about that, I don't handle the labels for this. It is an issue because of that downcase method, but it can also be a new feature to support camel case, depends on who looks at this. Either way, without removing that `downcase`, anyone using camel case will get an error as I've explained in the issue.

**Comment by dblock** [comment]
Excellent. I am merging this, it seems straightforward, cc: @LeFnord

**Comment by LeFnord** [comment]
👍 … and thanks again

**Comment by rayko** [comment]
Thanks for the attention :smile:

**Pull request merged**